tests/foreman/maintain/test_upgrade.py

```diff
@@ -146,11 +147,15 @@ def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
     )
     custom_host.download_repofile(product='satellite', release=last_y_stream)
     custom_host.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
-    # Install without system checks to get around installer checks
+    # Install with development tuning profile to get around installer checks
     custom_host.execute(
-        f'satellite-installer --scenario satellite --disable-system-checks --tuning {profile}',
+        'satellite-installer --scenario satellite --tuning development',
```

Comment:
is development supported for downstream or is this being used for negative scenario?

Reply:
```
# satellite-installer --scenario satellite --tuning
ERROR: Invalid option supplied for --tuning. Please choose from one of: default, medium, large, extra-large, extra-extra-large, development

See: 'satellite-installer --help'
[root@ip-10-0-168-174 ~]# satellite-installer --scenario satellite --tuning development
2023-09-12 05:03:04 [NOTICE] [root] Loading installer configuration. This will take some time.
2023-09-12 05:03:09 [NOTICE] [root] Running installer with log based terminal output at level NOTICE.
2023-09-12 05:03:09 [NOTICE] [root] Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG. See --full-help for definitions.
[...]
```